ui/perfherder/alerts/AlertTableRow.jsx

```diff
@@ -225,16 +226,22 @@ export default class AlertTableRow extends React.Component {
           </FormGroup>
         </td>
         <td className="px-0">
-          <span
-            className={starred ? 'visible' : ''}
+          <Button
+            color="black"
+            aria-label={
+              starred
+                ? 'Remove bookmark from this Alert'
+                : 'Bookmark this Alert'
+            }
+            className={`${bookmarkClass} border p-0 border-0 bg-transparent`}
```

Comment:
As there were too much code repetition, I thought it would be better to extract it into a `const` on line 201.

Reply:
It doesn't look like it's being used more than once - the `bookmarkClass` - but that's fine if you want to put it into a const.